dev-lang/cppfront/cppfront-0.7.4.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="A simpler and safer way to write C++ code"
+DESCRIPTION="Cppfront is a compiler from an experimental C++ 'syntax 2' \
+(Cpp2) to today's 'syntax 1' (Cpp1), to prove out some concepts, share \
+some ideas, and prototype features that can also be proposed for evolving \
+today's C++."
+HOMEPAGE="https://hsutter.github.io/cppfront/"
+COPYRIGHT="Herb Sutter"
+LICENSE="CppFront"
+REVISION="1"
+SOURCE_URI="https://github.com/hsutter/cppfront/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="028f44cc0cad26b51829e4abf7c5aedf8a31f852ab5dfbad54bb232f0a1d7447"
+PATCHES="cppfront-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cppfront$secondaryArchSuffix = $portVersion
+	cmd:cppfront = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:g++$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	g++ source/cppfront.cpp -std=c++20 -o cppfront
+}
+
+INSTALL()
+{
+	mkdir -p $binDir
+	cp cppfront $binDir
```

Comment:
This won't work with what PROVIDES declares, this way the binary ends up in /bin/x86 on 32bit resulting in:
```
	cmd:cppfront$secondaryArchSuffix = $portVersion
```
Better would be to install directly to /bin ($prefix/bin)
```diff
-	mkdir -p $binDir
-	cp cppfront $binDir
+	mkdir -p $prefix/bin
+	cp cppfront $refix/bin
```

Reply:
Being on X86_64 I will create a x86 VM otherwise I will keep not seeing this type of issues.
Will update/correct in the coming days.

Reply:
Thanks!